cluster-provision/images/vm-image-builder/README.md

```diff
@@ -73,3 +90,41 @@ $ kubectl apply -f <VMI yaml file>
 $ kubectl wait --for=condition=AgentConnected vmi $VMI_NAME --timeout 5m
 $ virtctl console testvm1
 ```
+
+### Build and publish multi-arch images
+# It does not support to build alpine-cloud-init, because the project
+# https://raw.githubusercontent.com/alpinelinux/alpine-make-vm-image/master/alpine-make-vm-image
+# does not support build Arm64 image.
```

Comment:
```suggestion
The multi-arch publish does not support building alpine-cloud-init because the [alpine-make-vm-image](https://raw.githubusercontent.com/alpinelinux/alpine-make-vm-image/master/alpine-make-vm-image) project does not support building Arm64 images.
```